editor/plugins/script_editor_plugin.cpp

```diff
@@ -352,9 +352,9 @@ void ScriptEditorQuickOpen::_notification(int p_what) {
 			connect("confirmed", callable_mp(this, &ScriptEditorQuickOpen::_confirmed));
 
 			search_box->set_clear_button_enabled(true);
-			[[fallthrough]];
-		}
-		case NOTIFICATION_VISIBILITY_CHANGED: {
```

Comment:
Worth noting that this was handling `VISIBILITY_CHANGED`. Does it change the behavior?

Reply:
I'll check, but I don't see anything special about this case, it seems like it should just be using `NOTIFICATION_THEME_CHANGED`